po/zh_CN.po

```diff
@@ -220,7 +220,7 @@ msgstr "正在下载 PKGBUILD..."
 
 #: src/download.rs:321
 msgid " PKGBUILDs up to date"
-msgstr " PKGBUILD 都是最新的"
+msgstr " PKGBUILDs 已经是最新的"
```

Comment:
https://github.com/Morganamilo/paru/pull/429#discussion_r659499857

https://github.com/ruanyf/document-style-guide/blob/master/docs/text.md#%E8%8B%B1%E6%96%87%E5%A4%84%E7%90%86

Reply:
feel free to just edit the pr

Reply:
> feel free to just edit the pr

I'm not a maintainer, so I think I can't directly edit it.